include/material/linear_elastic.tcc

```diff
@@ -41,7 +41,7 @@ Eigen::Matrix<double, 6, 6> mpm::LinearElastic<Tdim>::elastic_tensor() {
 template <unsigned Tdim>
 Eigen::Matrix<double, 6, 1> mpm::LinearElastic<Tdim>::compute_stress(
     const Vector6d& stress, const Vector6d& dstrain) {
-  
```

Comment:
Please don't add linear_elastic or material to this PR as this is only for Bingham model